quickwit/quickwit-doc-mapper/src/default_doc_mapper/mapping_tree.rs

```diff
@@ -640,6 +640,11 @@ fn value_to_bool(value: TantivyValue) -> Result<JsonValue, TantivyValue> {
             1 => Some(true),
             _ => None,
         },
+        TantivyValue::F64(number) => match number {
+            0.0 => Some(false),
+            1.0 => Some(true),
+            _ => None,
+        },
```

Comment:
i realized while writing the doc that missing this conversion was odd given u64/i64 had it, and bool->f64 was a thing

Reply:
Good catch.